### PR TITLE
Adds the Highlander gamemode

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Adds in a basic version of the highlander

Highlander can be triggered in three ways, event, gamemode and admin respawn.
Event and gamemode causes players to spawn as highlanders immediately; though the gamemode requires at least 5 players on for it to be active.

Highlanders on spawn get a pointer device that points towards the nuke disk and a claymore that has 100% chance to tear limbs apart for downed individuals.
Highlanders spawn with fast regen and semi-stun immunity; meaning that stun batons, soaps and tasers only make them drop their weapons briefly before the can get up again and fight and they can regen 6 brute damage across all body parts every second while their overall health is below 50.

Your goal as a highlander is to escape alive with the nuke disk on a shuttle, good luck.

### Changelog
CL: [New] - Adds highlanders
CL: [New] - Adds one new song to the Jukebox for true scotsmen that are drinking at the bar.
CL: [Improvement] - Trauma damage now only applies at Critical damage severity on the individual body part that received damage
CL: [Improvement] - Dismemberment now only works at max damage severity.
CL: [Fix] - Dismemberment only works when on the third level of slash damage now. Other methods such as door crushing and toolbox robusting (with the exception of bombs) no longer have the ability to gib any body part unless the body part was already mangled from the start.
CL: [Fix] - Enabled head beheading once again.

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
this should be the last thing, reviews pls

File: UnityProject/Assets/Scripts/Systems/Antagonists/Highlander.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ public class Highlander : Antagonist
 		public override void AfterSpawn(ConnectedPlayer player)
 		{
 			player.Script.playerHealth.EnableFastRegen();
+			player.Script.characterSettings.Speech = Speech.Scotsman;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/Antagonists/Highlander.cs:13]: just making sure, you want to force this being the only accent?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/Antagonists/Highlander.cs:13]: Yes

**Comment by MaxIsJoe** [comment]
wait, why le fok is the highlander music not working

**Comment by MaxIsJoe** [comment]
ok fixed audio

unsure why only one player keeps getting the disk objective while everyone else doesnt

**Comment by MaxIsJoe** [comment]
I am confused
image

Objective is not unique... but acts as one...
wat

**Comment by MaxIsJoe** [comment]
ok so.. apparently the escape objective **does** get added.. it just doesn't tell the players about it for some reason.

Good by me, please merge.

**Comment by corp-0** [comment]
what issue did you face with this? Before I merge

**Comment by MaxIsJoe** [comment]
> what issue did you face with this? Before I merge

Just the objective for some reason not showing for antags even though it's there

**Review by corp-0 — APPROVED** [review]

**Pull request merged**